### PR TITLE
feat: [CDS-42913]: schedule tab form basic

### PR DESCRIPTION
**Pull request by udayvunnam**
### Summary

freeze schedule tab basic form, TODO: validations and proper form sync

#### Screenshots

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 26 Issues 
* Code issue 26 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11608)

File: src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioUtil.ts [hunk 1]
```diff
@@ -81,7 +81,7 @@ export function isValidYaml(
 }
 
 export const getInitialValues = (freezeObj: any) => {
-  const pickedValues = pick(freezeObj, 'name', 'identifier', 'description', 'tags')
+  const pickedValues = pick(freezeObj, 'name', 'identifier', 'description', 'tags', 'windows')
```
Comment (ishantmehtaharness) [inline src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioUtil.ts:84]: Why do we need this one? Can we instead create `getInitialValueForScheduleSection`?

File: src/modules/10-common/utils/dateUtils.ts [hunk 2]
```diff
@@ -17,3 +17,599 @@ export const getReadableDateTime = (timestamp?: number, formatString = 'MMM DD,
   }
   return moment(timestamp).format(formatString)
 }
+
+export const ALL_TIME_ZONES = [
```
Comment (ishantmehtaharness) [inline src/modules/10-common/utils/dateUtils.ts:21]: Shall we remove it from CE codebase, and use this at that place?

Comment (udayvunnam) [reply src/modules/10-common/utils/dateUtils.ts:21]: Definitely. Thought we would merge this and raise a separate PR as the review from CE may take time

Comment (ishantmehtaharness) [reply src/modules/10-common/utils/dateUtils.ts:21]: Sounds good

**Review by ishantmehtaharness — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**